services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +45,47 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  error_code_t errCode;
```

Comment:
unused variable (compiler would give a warning about this so its good to check and resolve all compiler warnings)